package.json

```diff
@@ -14,8 +14,8 @@
     "clean": "rimraf ./lib ./types",
     "test": "jest",
     "lint": "tslint --project .",
-    "prettier:write": "prettier --write '{bin,src,test}/**/*.{js,ts}'",
-    "prettier": "prettier --check '{bin,src,test}/**/*.{js,ts}'",
+    "prettier:write": "prettier --write \"{bin,src,test}/**/*.{js,ts}\"",
```

Comment:
Any idea what happened with this change?  Doesn't seem like it's necessary 😃

Reply:
Ah, mentioned in another PR: https://github.com/sjbarag/brs/pull/247#discussion_r288729479

Nice catch!